redbot/core/core_commands.py

```diff
@@ -1229,70 +1231,67 @@ async def perms(self, ctx, level: int):
         await ctx.send("The new permissions level has been set.")
 
     @commands.command()
-    @commands.guild_only()
     @checks.is_owner()
-    async def leave(self, ctx: commands.Context):
-        """Leaves the current server."""
-        await ctx.send(_("Are you sure you want me to leave this server? (y/n)"))
+    async def leave(self, ctx: commands.Context, *server_ids: GuildConverter):
+        """Leaves a server.
+
+        If no server IDs are passed the local server will be left instead."""
+        guilds = list(server_ids)
+        if ctx.guild is None and not guilds:
+            return await ctx.send(_("You need to specify at least one server ID."))
+
+        leaving_local_guild = not len(guilds) >= 1
+
+        if leaving_local_guild:
+            guilds.append(ctx.guild)
+            msg = _("You haven't passed any server ID. Do you want me to leave this server? (y/n)")
+        else:
+            msg = _("Are you sure you want me to leave {} servers? (y/n)".format(len(guilds)))
```

Comment:
I think it would be safer to list the servers that the bot will live, especially that the user might pass them using their name which can be ambiguous.

Also:
- `.format()` needs to be called on the return value of `_()`, not on the string literal
- at Red, we use named format fields exclusively to help Translators
```suggestion
            msg = _("Are you sure you want me to leave {number} servers? (y/n)").format(
                number=len(guilds)
            )
```